week3/1-tuesday/class/exercise5/stacey-answers.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Exercise5</title>
+    </head>
+    <body>
+        <h1>Here's a couple ways you can express yourself</h1>
+        <hr>
+        <ul>
+            <li>If you want to sound normal, just use a regular li tag here</li>
+            <li>If you want to sound interesting, try the italics tag</li>
```

Comment:
Need the italics tag on this line.

Reply:
I added italics to line 11. Can you see this change?